### PR TITLE
Update TranqRotate.toc

### PR DESCRIPTION
**Pull request by FenixsRU**

### CODE REVIEW

File: TranqRotate.toc [hunk 1]
```diff
@@ -32,4 +33,4 @@ src\dragdrop.lua
 src\comms.lua
 src\defaults.lua
 src\settings.lua
-src\utils.lua
```
Comment (Slivo-fr) [inline TranqRotate.toc:35]: I don't understand this change, did I miss something ?

File: TranqRotate.toc [hunk 2]
```diff
@@ -32,4 +33,4 @@ src\dragdrop.lua
...
\ No newline at end of file
+src\utils.lua
```
Comment (Slivo-fr) [inline TranqRotate.toc:36]: I don't understand this change, did I miss something ?

Comment (FenixsRU) [reply TranqRotate.toc:36]: Sorry, accidentally deleted this line

**Comment by Slivo-fr** [comment]
Did the change on my side, thanks anyway

**Pull request closed without merging**